cpp/include/cudf/detail/utilities/device_operators.cuh

```diff
@@ -222,6 +222,16 @@ struct DeviceXor {
   }
 };
 
+/**
+ * @brief Operator for calculating Lead/Lag window function.
+ */
+template <bool is_lead>
```

Comment:
How is this is_lead being used ?

Reply:
It was used to distinguish `LEAD` from `LAG`. I've removed this, since this information is available in the `process_gpu_rolling()` function, from the `op` parameter.